### DISCUSSION TITLE
樱花的主页进不去

### DISCUSSION THREAD

**Discussion by ElspethED** [natfrp/wiki#95] [使用问题]
只有标题
标题点了都没反应
软件也是没有用

**Comment by ElspethED** [comment]
其实是加载不出来是我网的问题还是什么原因 前几天都可以用的

**Reply by Albsm** [reply]
AxiosError: Request failed with status code 400
遇到了同样的问题，昨晚发现无法在官网下载启动器，无法查看节点列表，dns已经设置成阿里公共dns

[a comment by an automated account is not shown]
